ca/config.go

```diff
@@ -45,7 +45,8 @@ const (
 type SecurityConfig struct {
 	mu sync.Mutex
 
-	rootCA *RootCA
+	rootCA     *RootCA
+	ExternalCA *ExternalCA
```

Comment:
I think this can be unexported. I don't see it being used outside the `ca` package, and I can't think of a good reason to expose it outside the package.